docker-compose/config/worker-ubuntu22-04.jsonnet

```diff
@@ -31,6 +80,9 @@ local common = import 'common.libsonnet';
       },
     }],
   }],
+  filePool: {
+    directoryPath: '/worker/filepool_cache',
```

Comment:
I do want to mention: please consider using a block device for this. The directory backend is mainly for debugging. Performance is pretty bad.

Reply:
converted to block device!